### PR TITLE
Stop prepending file:// to sourcemap paths

### PR DESCRIPTION
**Pull request by amcasey**
Submitting on behalf of @mhegazy.

### CODE REVIEW

**Comment by amcasey** [comment]
@sandersn was going to confirm that this didn't break some other debugger scenarios.

File: Jakefile.js [hunk 1]
```diff
@@ -321,9 +321,6 @@ function compileFile(outFile, sources, prereqs, prefixes, useBuiltCompiler, opts
             }
             else {
                 options += " -sourcemap";
-                if (!opts.noMapRoot) {
```
Comment (mhegazy) [inline Jakefile.js]: you can now remove all references to `noMapRoot` in this file.

**Comment by sandersn** [comment]
I'm about to try it with the Chrome/Ubuntu/jake combination to make sure that it works. (I believe this combination was broken before and that only gulp worked.)

I'll sign off when I confirm that it works.

**Comment by amcasey** [comment]
@sandersn I believe the gulp/jake difference is captured by https://github.com/Microsoft/TypeScript/issues/11105.

File: Jakefile.js (hunk 1 above)
Comment (amcasey) [inline Jakefile.js]: Oops, missed this one.  I'll create a separate PR.

Comment (amcasey) [inline Jakefile.js]: https://github.com/Microsoft/TypeScript/pull/16231

**Pull request merged**